components/server/src/authorization/defs.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export const OrganizationResourceType = "organization";
```

Comment:
Idea is for this file to largely contain the constants/types that we use throughout

Reply:
Works for me. :+1:

Reg. `defs.ts`: I find (slightly) longer names easier to memorize and type :right_anger_bubble:

Reply:
suggestion? `definitions`? `types`?

Reply:
done